frontend/src/Map.js

```diff
@@ -0,0 +1,69 @@
+
+
+
+
+
+
+
+
+
+
+
+// import React, { useState, useEffect } from "react";
+
+// function Map(props) {
+//   const [map, setMap] = useState(null);
+//   const [apiLoaded, setApiLoaded] = useState(false);
+//   useEffect(() => {
+//     let script = null;
```

Comment:
This file is entirely commented and if you export it, it will give an error. So how are we using this?